torch/onnx/_internal/fx/custom_operator.py

```diff
@@ -0,0 +1,58 @@
+from typing import Callable
+
+import torch
+from .function_dispatcher import _ATENLIB_FUNCTIONS, _OP_OVERLOAD_TO_EXPORTER_KEY_TABLE
+
+ONNX_CUSTOM_OP_LIB = torch.library.Library("onnx_custom", "DEF")
+
+
+def _register_onnx_custom_op_overload(schema: str):
+    """This function decorator registers the wrapped function as a `torch._ops.OpOverload` with the provided `schema`.
+
+    Arguments:
+        schema (str): The schema of the wrapped function in PyTorch's type system; e.g., "my_op(Tensor a, Tensor b) -> Tensor c".
+
+    Example:
+    The following example register the `a_funny_op` function as `torch.ops.onnx_custom.my_op` (type: torch._ops.OpOverload).
+
+        import torch
```

Comment:
Consider doctest format. Ref: https://docs.python.org/3/library/doctest.html, https://github.com/pytorch/pytorch/blob/436993d52bfca7a0676ef8f35fcdc16e5cbf1617/torch/onnx/verification.py#L1788

Enables example code to be validated by xdoctest locally and in CI (well, in near future).